packages/react/time-series/src/TimeSeries.tsx

```diff
@@ -203,6 +196,32 @@ export function TimeSeries(props: TimeSeriesProps) {
         }
       }
 
+      if (chartRef.current) {
+        const datasetRef = chartRef.current.data.datasets[0]
+
+        chartRef.current.data.labels = labels
+        datasetRef.data = values
+        if (chartRef.current.options.layout?.padding) {
+          chartRef.current.options.layout.padding = styles?.canvas?.padding
+        }
```

Comment:
We're missing a default value here I think, no?

```javascript
chartRef.current.options.layout.padding = styles?.canvas?.padding || defaultPadding
```

Reply:
Right